{
	"header": {
		"line": "CVE-2021-44476 (A sandboxing issue in Odoo Community 15.0 and earlier and Odoo Enterpr ...)",
		"name": "CVE-2021-44476",
		"description": "(A sandboxing issue in Odoo Community 15.0 and earlier and Odoo Enterpr ...)"
	},
	"annotations": [
		{
			"line": "\t{DSA-5399-1}",
			"type": "xref",
			"bugs": [
				"DSA-5399-1"
			]
		},
		{
			"line": "\t- odoo 16.0.0+dfsg.1-1 (bug #1035953)",
			"type": "package",
			"package": "odoo",
			"kind": "fixed",
			"version": "16.0.0+dfsg.1-1",
			"flags": [
				{
					"bug": 1035953
				}
			]
		},
		{
			"line": "\tNOTE: https://github.com/odoo/odoo/issues/107684",
			"type": "NOTE",
			"description": "https://github.com/odoo/odoo/issues/107684"
		},
		{
			"line": "\tNOTE: 14.0 patch at https://github.com/odoo/odoo/commit/be2c857a2e19b0a752555ab377ce5e1cb081a186",
			"type": "NOTE",
			"description": "14.0 patch at https://github.com/odoo/odoo/commit/be2c857a2e19b0a752555ab377ce5e1cb081a186"
		}
	]
}
